### DISCUSSION TITLE
How just work with one honeypot in hive sensor?

### DISCUSSION THREAD

**Discussion by r1beirin** [telekom-security/tpotce#1145] [Q&A]
Hello guys.

To a unique purpose I need just work with the Cowrie Honeypot. How can I disable other honeypots in my sensor to gain computing resources in my instance cloud?

Thanks!

**Comment by lcia-projects** [comment]
in \opt\tpot\etc there is a docker file called tpot.yml
its the basic configuration file for tpot and all its honeypots. I'm essentially wondering the same thingyou are. 

i *think* you can just comment out the honeypots you dont want to run.. but i'm afraid we'll need to adjust the firewall or some other files i dont know about. I cant seem to find much documentation or discussions about this... but the tpot.yml is where i'd start.
